joomla/jissues
# Add missing files for the label checking for in 3.x and 4.0

Pull Request for Issue # .

#### Summary of Changes

Add missing files for the label checking for in 3.x and 4.0

#### Testing Instructions

Apply the changes and add a PR changing the files. Check that they have the correct label.
